app/ui/lib/FieldLabel.tsx

```diff
@@ -17,6 +17,7 @@ interface FieldLabelProps {
   tip?: string
   optional?: boolean
   className?: string
+  hideOptionalTag?: boolean
```

Comment:
The prop can be removed from this component thanks to 5e515cbf4e28a33386f3492485356ce71584aa59. It's unused because `optional` covers it.